src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -102,6 +100,22 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 		this.updatePropertiesTableColumnHeaders();
 	}
 
+	private static getTopOperationLabel(target: string): string {
```

Comment:
These don't really need to be a class member - I'd just put them at the top level as a module-level function (keeps classes as small and simple as possible). But that's up to you if you prefer it this way